core/eolearn/core/eodata.py

```diff
@@ -185,10 +187,10 @@ def _parse_feature_value(self, value: object, feature_name: str) -> gpd.GeoDataF
             value = gpd.GeoDataFrame(geometry=value, crs=value.crs)
 
         if isinstance(value, gpd.GeoDataFrame):
-            if self.feature_type is FeatureType.VECTOR and FeatureType.TIMESTAMP.value.upper() not in value:
+            if self.feature_type is FeatureType.VECTOR and TIMESTAMP_COLUMN not in value:
                 raise ValueError(
-                    f"{self.feature_type} feature has to contain a column 'TIMESTAMP' with timestamps but "
-                    f"feature {feature_name} doesn't not have it."
+                    f"{self.feature_type} feature has to contain a column '{TIMESTAMP_COLUMN}' with timestamps but "
+                    f"feature {feature_name} does not not have it."
                 )
```

Comment:
this will break any old EOPatches right?

Reply:
no, the `TIMESTAMP_COLUMN` value remained the same (`"TIMESTAMP"`) since it's more in line with the dataframe format.

Reply:
right! thanks :see_no_evil: